pages/index.js

```diff
@@ -26,28 +110,95 @@ export default function Home() {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-   <Flex gap="300px" justify="space-between" align="center">
-   <MenuBar/>
-        <Box  m="10" justifyItems="center" h="36px">
-      <Text>Signed in as {session.user.email} <br /></Text>
-      <Button onClick={() => signOut()} h="25px">Sign out</Button>
+      <Flex
+        bgColor="purple.50"
+        justify="space-between"
+        align="center"
+        height="70px"
+      >
+        <MenuBar />
+        <Box m="10" justifyItems="center" h="36px">
+          <Text>Signed in as {session.user.email}</Text>
+          <Button onClick={() => signOut()} h="25px">
+            Sign out
+          </Button>
+        </Box>
+      </Flex>
+      <Box css={backgroundImgStyle}>
+        <Container centerContent="true">
+          <Box p="5px" rounded="md" mb="55px" bgColor="purple.900/80">
+            <Text color="white" textStyle="3xl" fontWeight="bold">
+              Welcome to favorite cities website!
+              <br />
+            </Text>
+          </Box>
+          <Text mt="3.5" css={listStyle} textStyle="lg" fontWeight="medium">
+            In order to explore the cities of the world{" "}
+            <Link
+              _hover={{ color: "white" }}
+              variant="underline"
+              color="purple.700"
+              href="/search"
+            >
+              search here
+            </Link>
+            !
+          </Text>
+          <Flex wrap="wrap" gap="100px">
+            <Box css={listStyle}>
+              <Center>
+                <Heading size="xl">Favorite cities 💜</Heading>
+              </Center>
+              <Center>
+                <Text>{noFavoriteCitiesText}</Text>
+              </Center>
+              {favoriteCities.length > 0 &&
+                favoriteCities[0].map((city) => {
+                  return (
+                    <Center>
+                      <Box as="ul">
+                        <li key={city.cityName}>
+                          City: {city.cityName} | Country: {city.country}
+                        </li>
+                        <br></br>
+                      </Box>
+                    </Center>
+                  );
+                })}
+            </Box>
+            <Box css={listStyle}>
+              <Center>
+                <Heading size="xl">Random cities </Heading>
+              </Center>
+              {fetchedRandomCity.length > 0 &&
+                fetchedRandomCity.map((city) => {
+                  return (
+                    <Center>
```

Comment:
// The `key` property should be moved from the `<Box>` element to the `<Center>` component here for better React optimization.